client/sections.js

```diff
@@ -5,6 +5,13 @@ const fs = require( 'fs' ); // eslint-disable-line import/no-nodejs-modules
 const path = require( 'path' ); // eslint-disable-line import/no-nodejs-modules
 
 const sections = [
+	{
+		name: 'root-entry',
```

Comment:
Revisiting this again, `root-entry` shouldn't be `entry` at all. I think we should keep it just `root`. The fact that it's not its own entry point but it's a section, and we name it an entry here could add unnecessary confusion.

Reply:
I've changed this to root.

Reply:
We'll likely have a test to update with that change, too.